client/my-sites/patterns/components/get-started/index.tsx

```diff
@@ -47,7 +47,11 @@ export function PatternsGetStarted() {
 					<img
 						className="patterns-get-started__item-image"
 						src={ imageBlockPatterns }
-						alt=""
+						alt={ translate( "A support page with the title 'Use block patterns'", {
```

Comment:
Maybe that's just me but I would be a bit more direct:
```suggestion
						alt={ translate( "Support page with the title 'Use block patterns'", {
```